src/actions/tags.js

```diff
@@ -0,0 +1,12 @@
+let nextTagId = 6
+
+export const addTag = (categoryId, value) => {
+  return {
+    payload: {
+      id: nextTagId++,
+      categoryId: categoryId,
+      value: value
```

Comment:
when object property name  and value have same name, you can simplify it by only write property name. like this 
`payload: {
         id: nextTagId++,
         categoryId,
         value
}`